include/SFML/Graphics/Text.hpp

```diff
@@ -145,6 +145,36 @@ class SFML_GRAPHICS_API Text : public Drawable, public Transformable
     void setCharacterSize(unsigned int size);
 
     ////////////////////////////////////////////////////////////
+    /// \brief Set the additional line spacing offset
+    ///
+    /// The spacing between lines is defined by the font.
```

Comment:
Maybe add "default" to all of these similar comments ,e.g.

`The default spacing between lines is defined by the font.`